source/android/Foundational_Components_Kernel.rst

```diff
@@ -2,125 +2,117 @@
 Android Kernel
 ##############
 
+Android is built around a Linux kernel. By default, Android's
+``boot.img`` is build from a binary kernel Image located in::
+
+   ${YOUR_PATH}/ti-aosp-15/device/ti/am62x-kernel/kernel/6.6
+
```

Comment:
Please do not use the old code-block notation.

```suggestion
``boot.img`` is build from a binary kernel Image located in:

.. code-block:: text

   ${YOUR_PATH}/ti-aosp-15/device/ti/am62x-kernel/kernel/6.6

```

Reply:
Ok, thanks. Will fix in v3.